apps/api/drizzle.config.ts

```diff
@@ -19,13 +19,13 @@
 
 import { defineConfig } from 'drizzle-kit';
 
-const PGDATABASE = process.env.PGDATABASE;
-const PGUSER = process.env.PGUSER;
-const PGPASSWORD = process.env.PGPASSWORD;
-const PGHOST = process.env.PGHOST;
+const PG_DATABASE = process.env.PG_DATABASE || 'postgres';
+const PG_USER = process.env.PG_USER || 'postgres';
+const PG_PASSWORD = process.env.PG_PASSWORD || 'password';
+const PG_HOST = process.env.PG_HOST || 'localhost:5432';
```

Comment:
inclined to suggest we don't offer defaults/fallback (i.e. leave them as empty strings), and error out instead if these are missing (i.e. if string.length < 1)

Reply:
Yes, that is a great point

Reply:
OK great, removed